crypto/fipsmodule/bn/exponentiation.c

```diff
@@ -124,11 +124,13 @@
      defined(OPENSSL_OPENBSD) || defined(OPENSSL_FREEBSD)) &&                        \
     defined(OPENSSL_AARCH64)
 
-#include "../../../third_party/s2n-bignum/include/s2n-bignum_aws-lc.h"
+#include "../../../third_party/s2n-bignum/s2n-bignum_aws-lc.h"
 
 #define BN_EXPONENTIATION_S2N_BIGNUM_CAPABLE 1
 
-OPENSSL_INLINE int exponentiation_use_s2n_bignum(void) { return 1; }
+OPENSSL_INLINE int exponentiation_use_s2n_bignum(void) {
+  return CRYPTO_is_NEON_capable();
```

Comment:
is this moving the capability checks in `crypto/fipsmodule/bn/montgomery.c` to here instead?

Reply:
Exactly :)

Reply:
Scratch that. They are guarding two different things :)